enarx-keep-sev-shim/src/hostcall.rs

```diff
@@ -85,7 +85,7 @@ impl<'a> HostCall<'a> {
     /// The parameters returned can't be trusted.
     pub unsafe fn write(&mut self, fd: usize, bytes: &[u8]) -> Result<usize, libc::c_int> {
         let cursor = self.0.cursor();
-        let buf = cursor.copy_slice(bytes).or(Err(libc::EMSGSIZE))?;
+        let (_c, buf) = cursor.copy_slice(bytes).or(Err(libc::EMSGSIZE))?;
```

Comment:
```suggestion
        let (_, buf) = cursor.copy_slice(bytes).or(Err(libc::EMSGSIZE))?;
```

Do that everywhere please.

Reply:
I hate doing this just to satisfy rustfmt... ah well..